api/gateways/index.js

```diff
@@ -16,11 +16,13 @@ const searchPropertiesGateway = require('./SearchPropertiesGateway')(apiRequeste
 const availableAppointmentsGateway = require('./AvailableAppointmentsGateway')(apiRequester.makeGetRequest);
 const saveRepairGateway = require('./SaveRepairGateway')(apiRequester.makePostRequest);
 const repairTriageGateway = require('./RepairTriageGateway')(apiRequester.makeGetRequest);
+const getCommunalPropertyRepairsGateway = require('./GetCommunalPropertyRepairsGateway')(apiRequester.makeGetRequest);
```

Comment:
The gateway should not have an API verb prefixed the name, the API verb could be on the method/function. Remove the API verb from the gateway name, i.e. `GetCommunalPropertyRepairsGateway` -> `CommunalPropertyRepairsGateway`

Reply:
The above comment hasn't been addressed.